src/Controller/PagesController.php

```diff
@@ -28,6 +30,8 @@
  */
 class PagesController extends AppController
 {
+    const PATH_CAROUSEL = "../webroot/img/carousel/";
+
```

Comment:
http://book.cakephp.org/3.0/en/core-libraries/global-constants-and-functions.html#core-definition-constants